<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestConceptDialog</class>
 <widget class="QDialog" name="QtTestConceptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Concept</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_load">
       <property name="text">
        <string>&amp;Load test concept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_test"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>651</width>
        <height>518</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="widget_left" native="true">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>500</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_right" native="true">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>500</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
